src/Raven.Server/Smuggler/Documents/DatabaseDestination.cs

```diff
@@ -2174,14 +2174,14 @@ public Stream GetTempStream()
             }
         }
 
-        private class SubscriptionActions : ISubscriptionActions
+        protected class SubscriptionActions : ISubscriptionActions
```

Comment:
I don't think you need to change anything in this class... besides this change..